src/components/shared/Button/CommandBarButton.css

```diff
@@ -27,9 +27,8 @@
   color: var(--theme-body-color);
 }
 
-.command-bar-button > * {
```

Comment:
Is this the only icon affected by the rule?

Reply:
yes, I didn't find any other icon which uses this class name